shiny/Data-Ingest/server_files/ingest_workflow_svr.R

```diff
@@ -118,29 +118,31 @@ output$input_record_df <- renderPrint({Shared.data$input_record_df})
 
 
 ######### Formats Svr #############
-
-## Output List ##
-FormatRecordList <- list()
-
 output$autoname <- renderPrint({Shared.data$selected_row}) #_local
 
 ######### Mimetype Name ##################
 updateSelectizeInput(session, "MimetypeName", choices = mimetypes, server = TRUE)
 
 observeEvent(input$createFormatRecord, {
+  ## Output List ##
+  FormatRecordList <<- list()
   ## MimetypeID
   FormatRecordList$mimetypeName <- input$MimetypeName
   FormatRecordList$mimetypeID <- mimetype_sub %>% dplyr::filter(type_string %in% FormatRecordList$mimetypeName) %>% pull(id)
   
   ## Everything else
-  FormatRecordList$NewMimeType <- input$NewMimeType
-  FormatRecordList$NewFormatName <- input$NewFormatName
-  FormatRecordList$HeaderBoolean <- input$HeaderBoolean
-  FormatRecordList$SkipLines <- input$SkipLines #This should appear only if header = TRUE
-  FormatRecordList$FormatNotes <- input$FormatNotes
+  FormatRecordList$NewMimeType <- "text/csv" #input$NewMimeType
+  FormatRecordList$NewmimetypeID <- mimetype_sub %>% dplyr::filter(type_string %in% FormatRecordList$NewMimeType) %>% pull(id)
+  FormatRecordList$NewFormatName <- "Test_Format_Name_99" #input$NewFormatName
+  FormatRecordList$HeaderBoolean <- "TRUE" #ifelse((input$HeaderBoolean == "Yes"), "TRUE", "FALSE")
+  FormatRecordList$SkipLines <- "3" #input$SkipLines #This should appear only if header = TRUE
+  FormatRecordList$FormatNotes <- "These are some test notes" #input$FormatNotes
```

Comment:
Looks like all of these are still set at testing values.

Reply:
Got it. Good catch. Fixing this uncovered a bug that took me over an hour to squash!